dist/get.kobman.io

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-sudo dpkg --configure -a
 #Install: stable
 
 # Global variables
 export KOBMAN_VERSION="0.01"
```

Comment:
change to. ,  
export KOBMAN_VERSION=${KOBMAN_VERSION:-"0.0.1"}

@asa1997 Also update in the tmpl get.kobman.io as  export KOBMAN_VERSION=${KOBMAN_VERSION:-@KOBMAN_VERSION@}

Reply:
Sure Vinod

Reply:
Modified